code/datums/components/nobeaconcrossing.dm

```diff
@@ -0,0 +1,43 @@
+/datum/component/no_beacon_crossing
+	var/atom/parentatom
```

Comment:
Why do people keep trying to do this, stop it. Typecast when you need it don't store your parent again